runtime/domain.c

```diff
@@ -1577,19 +1577,18 @@ int caml_try_run_on_all_domains_async(
 
 void caml_interrupt_self(void)
 {
-  interrupt_domain(&domain_self->interruptor);
+  interrupt_domain_local(Caml_state);
 }
 
 /* async-signal-safe */
```

Comment:
What does `async-signal-safe` mean? Is it defined somewhere?

Reply:
This is POSIX terminology, see for instance `man signal-safety`

Reply:
Could the comment include the `man signal-safety` advice?